src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.coffee

```diff
@@ -27,7 +27,7 @@ DashboardCompanySelectboxCtrl = ($scope, $state, $stateParams, $uibModal, MnoeCu
     selectBox.close()
 
   selectOrganization = ->
-    selectBox.organization = _.find(MnoeCurrentUser.user.organizations, { id: MnoeOrganizations.selectedId })
+    selectBox.organization = MnoeOrganizations.getSelected().organization || _.find(MnoeCurrentUser.user.organizations, { id: MnoeOrganizations.selectedId })
```

Comment:
I think it would be safer to use a navigation operator here:
`MnoeOrganizations.getSelected()?.organization `